### DISCUSSION TITLE
harnessing multiple cores in compilation

### DISCUSSION THREAD

**Discussion by smatkovi** [geobra/harbour-shmoose#89] [Ideas]
i would suggest when compiling in the sdk to replace ./scons with ./scons -j$(nproc) to harness all cores, it takes a really long time to compile swift otherwise

**Comment by geobra** [comment]
Indeed. This will speed up the scons build. But keep in mind that this also uses more resources on the host. I always run in errors within github actions, if I use all the available cores.
